proxy/src/project.rs

```diff
@@ -1,27 +1,12 @@
-//! Abstractions for projects returned by the API.
+//! Abstractions for projectsI.
```

Comment:
```suggestion
//! Abstractions for projects.
```

Could you expand that comment? What abstractions? Why do we need them?